modules/ROOT/pages/cassandra/cassandra-use-cases.adoc

```diff
@@ -0,0 +1,231 @@
+= Use Cases - Mule 4
```

Comment:
Again--these are not titles containing text that a customer would search for in Google.

Reply:
If it's a file of examples, call them examples in the title.

Reply:
Hopefully you got Meera to change this.